.circleci/config.yml

```diff
@@ -23,29 +23,33 @@ jobs:
 
     steps:
       - samvera/cached_checkout
+
+      - run: 'sudo apt-get update'
+
       - samvera/bundle:
           ruby_version: << parameters.ruby_version >>
           bundler_version: << parameters.bundler_version >>
           cache_version: "2"
+
       # - samvera/rubocop
+
       - samvera/parallel_rspec
 
 workflows:
-  version: 2
   ci:
     jobs:
-      # - test:
+      # - bundle_and_test:
       #     name: "ruby3-0"
       #     ruby_version: "3.0.0"
-      - test:
+      - bundle_and_test:
           name: "ruby2-7"
-          ruby_version: "2.7.0"
-      - test:
+          ruby_version: "2.7.5"
+      - bundle_and_test:
           name: "ruby2-6"
-          ruby_version: "2.6.5"
-      - test:
+          ruby_version: "2.6.9"
+      - bundle_and_test:
           name: "ruby2-5"
-          ruby_version: "2.5.7"
-      - test:
-          name: "ruby2-4"
-          ruby_version: "2.4.9"
+          ruby_version: "2.5.9"
+      #- bundle_and_test:
+      #    name: "ruby2-4"
+      #    ruby_version: "2.4.9"
```

Comment:
How about removing the 2.4 job entirely instead of commenting it out?

Reply:
Certainly.